src/Infrastructure.EntityFramework/Configurations/ProjectEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Bit.Infrastructure.EntityFramework.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+public class ProjectEntityTypeConfiguration : IEntityTypeConfiguration<Project>
+{
+    public void Configure(EntityTypeBuilder<Project> builder)
+    {
+        builder
+            .Property(s => s.Id)
+            .ValueGeneratedNever();
+
+        builder
+            .HasKey(s => s.Id)
+            .IsClustered();
+
+        builder
+            .HasIndex(s => s.DeletedDate)
+            .IsClustered(false);
+
+        builder
+            .HasIndex(s => s.OrganizationId)
+            .IsClustered(false);
```

Comment:
The indexes for these are not reflected in the sql server migration. Please run the sql server migration, @Thomas-Avery can assist with that.